### ISSUE TITLE
Master branch specs are failing with I18n::InvalidLocale

### ISSUE THREAD

**Issue by andyw8** [thoughtbot/formulaic#44]
When I run the specs locally I get one failure:

    Failures:

      1) Fill in user form finds and fills a spanish locale date field
         Failure/Error: I18n.locale = :es
         I18n::InvalidLocale:
           :es is not a valid locale
         # ./spec/features/fill_in_user_form_spec.rb:88:in `block (2 levels) in <top (required)>'

Master passed 'about a month ago' on Travis, so I assume a gem release since then has caused this breakage. I can't the inspect the `Gemfile.lock` created by the CI build to compare against my local version, so I'm not sure how to track down the cause of this.

**Comment by calebthompson** [comment]
Here's my Gemfile.lock. It passes all specs:

```ruby
PATH
  remote: .
  specs:
    formulaic (0.1.3)
      activesupport
      capybara
      i18n

GEM
  remote: https://rubygems.org/
  specs:
    activesupport (4.1.4)
      i18n (~> 0.6, >= 0.6.9)
      json (~> 1.7, >= 1.7.7)
      minitest (~> 5.1)
      thread_safe (~> 0.1)
      tzinfo (~> 1.1)
    capybara (2.4.1)
      mime-types (>= 1.16)
      nokogiri (>= 1.3.3)
      rack (>= 1.0.0)
      rack-test (>= 0.5.4)
      xpath (~> 2.0)
    coderay (1.1.0)
    diff-lcs (1.2.5)
    i18n (0.6.11)
    json (1.8.1)
    method_source (0.8.2)
    mime-types (2.3)
    mini_portile (0.6.0)
    minitest (5.4.0)
    nokogiri (1.6.3.1)
      mini_portile (= 0.6.0)
    pry (0.10.0)
      coderay (~> 1.1.0)
      method_source (~> 0.8.1)
      slop (~> 3.4)
    rack (1.5.2)
    rack-test (0.6.2)
      rack (>= 1.0)
    rake (10.3.2)
    rspec (3.0.0)
      rspec-core (~> 3.0.0)
      rspec-expectations (~> 3.0.0)
      rspec-mocks (~> 3.0.0)
    rspec-core (3.0.3)
      rspec-support (~> 3.0.0)
    rspec-expectations (3.0.3)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.0.0)
    rspec-mocks (3.0.3)
      rspec-support (~> 3.0.0)
    rspec-support (3.0.3)
    slop (3.6.0)
    thread_safe (0.3.4)
    tzinfo (1.2.1)
      thread_safe (~> 0.1)
    xpath (2.0.0)
      nokogiri (~> 1.3)

PLATFORMS
  ruby

DEPENDENCIES
  bundler (~> 1.3)
  formulaic!
  pry
  rake
  rspec
```

**Comment by andyw8** [comment]
That seems to be for v0.1.3, could you regenerate it for v0.2.0 and I will compare with mine?

**Comment by andyw8** [comment]
I played around a bit more, and found that it passes intermittently. It looks like something the specs is order-dependent, since `rspec --seed 31845` with [this Gemfile.lock](https://gist.github.com/andyw8/f5d2a938241e5d1ded21) fails every time.

**Comment by KevinSjoberg** [comment]
@calebthompson @andyw8 fixed in #53

**Comment by KevinSjoberg** [comment]
@calebthompson since #53 is merged. We can close this issue.
